### ISSUE TITLE
Acceptance Testing: Google Maps

### ISSUE THREAD

**Issue by mavistoh** [CS2103AUG2017-T10-B3/main#77]
<img width="1280" alt="screen shot 2017-11-03 at 2 23 52 pm" src="https://user-images.githubusercontent.com/31090786/32363150-c76a748e-c0a8-11e7-92b9-72c9b8d974b3.png">

Google maps characters invalid.

**Comment by AceCentury** [comment]
Hi there, this issue seems to be isolated to Mac users only. Our team member who is using a Mac also has this issue.

**Comment by yuheng222** [comment]
Yup, this issue seems to be faced by Mac users only due to font incompatibility. We have tried changing the font in the browser panel to a compatible one but to no avail since the issue seems to stem from the Google Map search engine itself.
